tests/phpunit/TestFramework/PhpUnit/Adapter/PhpUnitAdapterTest.php

```diff
@@ -323,6 +377,23 @@ public static function executionOrderProvider(): iterable
         yield [true, '13.0'];
     }
 
+    #[DataProvider('coverageWithoutSourceProvider')]
+    public function test_supports_coverage_without_source(bool $expected, string $version): void
```

Comment:
for the test itself, I think the reverse order makes more sense. For the assertions themselves I think it's more muscle memory than anything at this point, but for the test input it messes me up too hard 😅